tools/scripts/script.ps1

```diff
@@ -0,0 +1,193 @@
+    }
+}
+
+function Uninstall-AIShellApp {
+    $destination = $Script:InstallLocation
+    if (Test-Path $destination) {
+        if ($IsWindows) {
+            Remove-Item -Path $destination -Recurse -Force -ErrorAction Stop
+
+            # Update the user-scope Path env variables to remove AIShell.
+            $userPath = [Environment]::GetEnvironmentVariable('Path', [EnvironmentVariableTarget]::User)
+            if ($userPath.Contains($destination)) {
+                Write-Host "[Removing AIShell from user-scope Path]" -ForegroundColor Green
+                $newUserPath = $userPath.Split(';', [StringSplitOptions]::RemoveEmptyEntries -bor [StringSplitOptions]::TrimEntries) |
+                    Where-Object { $_ -ne $destination } |
+                    Join-String -Separator ';'
+                [Environment]::SetEnvironmentVariable("Path", $newUserPath, [EnvironmentVariableTarget]::User)
+                Write-Host "[AIShell removed from user-scope PATH]" -ForegroundColor Green
+            }
+        } else {
+            sudo rm -rf $destination
+            if ($LASTEXITCODE -ne 0) {
+                throw "Failed to remove the AIShell app from '$destination'."
+            }
+
+            $symlink = $Script:MacSymbolicLink
+            sudo rm $symlink
+            if ($LASTEXITCODE -ne 0) {
+                throw "Failed to remove the symbolic link '$symlink'."
+            }
+        }
+        Write-Host "[AIShell removed from '$destination']" -ForegroundColor Green
+    } else {
+        Write-Host "[AIShell cannot be found at '$destination', skip removing]" -ForegroundColor Yellow
+    }
+}
+
+function Install-AIShellModule {
+    Write-Host "[Installing the PowerShell module 'AIShell' ...]`n" -ForegroundColor Green
+    Install-PSResource -Name AIShell -Repository PSGallery -Prerelease -ErrorAction Stop
+    Write-Host "[AIShell Module installed]" -ForegroundColor Green
+}
+
+function Uninstall-AIShellModule {
+    if (Get-InstalledPSResource -Name "AIShell") {
+        try {
+            Write-Host "[Uninstalling AIShell Module ...]" -ForegroundColor Green
+            Uninstall-PSResource -Name AIShell -ErrorAction Stop
+            Write-Host "[AIShell Module uninstalled]" -ForegroundColor Green
+        } catch {
+            throw "Failed to uninstall the 'AIShell' module. Please check if the module got imported in any active PowerShell session. If so, please exit the session and try this script again."
+        }
+    } else {
+        Write-Host "[AIShell Module cannot be found, skip uninstalling]" -ForegroundColor Yellow
+    }
+}
+
+<###################################
+#
+#           Setup/Execute
+#
+###################################>
+
+Resolve-Environment
+
+if ($Uninstall) {
+    Uninstall-AIShellApp
+} else {
+    Install-AIShellApp
+}
+
+Write-Host "`n----------------------------------------`n" -ForegroundColor Green
+
+if ($Uninstall) {
+    Uninstall-AIShellModule
+} else {
+    Install-AIShellModule
```

Comment:
If Mac, "Currently the AIShell PowerShell module will only work in iTerm2 terminal and still has limited support but if you would like to test it, you can install it with `Install-PSResource -Name AIShell -Repository PSGallery -Prerelease`. The aish executable is added to your path, please run aish to use the standalone experience"

Reply:
Resolved, but not tested.

Reply:
Tested and confirmed working